ffi/posix_h.lua

```diff
@@ -17,6 +17,7 @@ static const int ETIME = 62;
 static const int EAGAIN = 11;
 static const int EINVAL = 22;
 static const int EPIPE = 32;
+static const int ENODEV = 19;
```

Comment:
Needs a matching declaration if `ffi-cdecl/posix_decl.c` so that it doesn't get lost the next time I refresh those ;).

Reply:
And should probably be sorted above EINVAL, too.

Reply:
Added a declaration there and sorted them.